cli/k8sflex/flexrex

```diff
@@ -0,0 +1,190 @@
+#!/bin/bash
+
+# Copyright 2016 Dell EMC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Notes:
+#  - Please install "jq" package before using this driver.
+#  - Please install and configure REX-Ray before using this driver
+usage() {
+	err "Invalid usage. Usage: "
+	err "\t$0 init"
+	err "\t$0 attach <json params>"
+	err "\t$0 detach <mount device>"
+	err "\t$0 mount <mount dir> <mount device> <json params>"
+	err "\t$0 unmount <mount dir>"
+	exit 1
+}
+
+err() {
+	echo -ne $* 1>&2
+}
+
+log() {
+	echo -ne $* >&1
+}
+
+ismounted() {
+	MOUNT=`findmnt -n ${MNTPATH} 2>/dev/null | cut -d' ' -f1`
+	if [ "${MOUNT}" == "${MNTPATH}" ]; then
+		echo "1"
+	else
+		echo "0"
+	fi
+}
+
+attach() {
+	VOLUMEID=$(echo $1 | jq -r '.volumeID')
+	if [ -z $VOLUMEID ]; then
+		err "{\"status\": \"Failure\", \"message\": \"Unable to extract volumeID\"}"
+		exit 1
+	fi
+
+	OUTPUT=$(rexray volume attach ${VOLUMEID} --format json)
```

Comment:
I guess I don't need to store the command results here. Originally I was using the output to look for the device, but I have to make a second call to get it.  device info doesn't come back from the attach command -- it can't because that relies on the localdevice headers, which are gathered/sent *before* the attach (obviously). so no mapping can be done.